packages/experimental/Switch/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@fluentui-react-native/experimental-switch",
```

Comment:
We can just use "switch" if that package name isn't used yet. Saves some headache of changing the name later